components/Post/PostContent.tsx

```diff
@@ -53,6 +53,9 @@ const components: Components<{
 
 const PostContent = ({ post }: Props) => {
   const description = getFirstParagraph(post.body);
+  const config = useAppSelector((state) => state.tinaData.main_config);
+  const hostUrl = useAppSelector((state) => state.hostUrl.link);
```

Comment:
```suggestion
  const hostURL = useAppSelector((state) => state.hostUrl.link);
```

This is an abbreviation

![image](https://github.com/1alexvash/AlexCodeBlog/assets/46556479/750046e4-5bfd-45c0-8912-e95b3d74be62)